.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,15 @@
-Thank you for sending the PR! We appreciate you spending the time to work on these changes.
-Help us understand your motivation by explaining why you decided to make this change:
+<!-- Thanks for submitting a pull request! We appreciate you spending the time to work on these changes. Please provide enough information so that others can review your pull request. The three fields below are mandatory. -->
 
+## Summary
 
-Changelog:
-----------
+<!-- Explain the **motivation** for making this change. What existing problem does the pull request solve? -->
 
-Help reviewers and the release process by writing your own changelog entry. See http://facebook.github.io/react-native/docs/contributing#changelog for an example.
+## Changelog
 
-[CATEGORY] [TYPE] - Message
+<!-- Help reviewers and the release process by writing your own changelog entry. See http://facebook.github.io/react-native/docs/contributing#changelog for an example. -->
 
+[CATEGORY] [TYPE] - Message
 
-Test Plan:
-----------
+## Test Plan
```

Comment:
Don't drop the colon after "plan", to ensure Phabricator pulls this into the Test Plan field.